cmake/std/atdm/apps/empire/EMPIRETrilinosEnables.cmake

```diff
@@ -1,4 +1,21 @@
+IF ("$ENV{ATDM_CONFIG_BUILD_TYPE}" STREQUAL "DEBUG")
+	ATDM_SET_ENABLE(PanzerAdaptersSTK_CurlLaplacianExample-ConvTest-Quad-Order-4_DISABLE ON)
+	ATDM_SET_ENABLE(PanzerAdaptersSTK_MixedPoissonExample-ConvTest-Hex-Order-3_DISABLE ON)
+ENDIF()
+
+IF (ATDM_USE_CUDA)
+	ATDM_SET_ENABLE(Sacado_ENABLE_HIERARCHICAL_DFAD ON)
+ENDIF()
+
 ATDM_SET_ENABLE(MueLu_ENABLE_Kokkos_Refactor_Use_By_Default ON)
+ATDM_SET_ENABLE(Panzer_ENABLE_EXAMPLES ON)
```

Comment:
Since one important use case is to just build and install libraries, it might be better if we keep `Panzer_ENABLE_EXAMPLES=ON` and `Panzer_ENABLE_TESTS=ON` out of this file.  If one wants to run Panzer tests, that is fine but it is not needed to just install Trilinos for EMPIRE to use.

So can these two options stay in the BuildScripts script?